tracetools_test/tracetools_test/case.py

```diff
@@ -78,7 +83,10 @@ def setUp(self):
         # Get timestamp before trace (ns)
         timestamp_before = int(time.time() * 1000000000.0)
 
-        exit_code, full_path = run_and_trace(
+        # Set trace test ID env var for spawned processes
+        trace_test_id = get_trace_test_id(self._session_name_prefix)
+        os.environ[TRACE_TEST_ID_ENV_VAR] = trace_test_id
+        self._exit_code, self._full_path = run_and_trace(
```

Comment:
This is just nitpicking, but it kind of looks like `self._exit_code` could just be a stack variable, rather than a class member.  That is, it never seems to be used outside of `__init__` here.

Reply:
Good point, I switched to a local variable: https://github.com/ros2/ros2_tracing/compare/cdab11e618ce64a089bdd391699da96894536d55..edeaa9454d15e05865b08d625f15e2c7ecbb8dc7.